_data/navbars/getting-started.yml

```diff
@@ -119,6 +119,17 @@ section:
       path: /getting-started/kubernetes/hardway/istio-integration
   - title: System requirements
     path: /getting-started/kubernetes/requirements
+  - title: VPP dataplane tech preview
```

Comment:
VPP dataplane is fine here.
In the rest of the docs, let's use VPP dataplane as well (instead of Calico VPP dataplane or Calico-VPP dataplane. This mirrors what we did for eBPF.

Use eBPF in all occurrences.